kvmagent/kvmagent/plugins/virtiofs_plugin.py

```
@@ -0,0 +1,425 @@
+            domain, conn = get_vm_domain(cmd.vmInstanceUuid)
+
+            # 3. Build virtiofs XML
+            xml_str = build_virtiofs_xml(cmd.tag, cmd.sourcePath)
+
+            # 4. Attach device to running VM (hot-plug)
+            domain.attachDeviceFlags(xml_str, libvirt.VIR_DOMAIN_AFFECT_LIVE)
+
+            # 5. Also attach to config for persistence (VM restart)
+            try:
+                domain.attachDeviceFlags(xml_str, libvirt.VIR_DOMAIN_AFFECT_CONFIG)
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)